config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+
+  root 'welcome#index'
   resources :dashboard, only: :index
+  get '/auth/google_oauth2/callback', to: 'sessions#create'
```

Comment:
post '/auth/:provider/callback', to: 'sessions#create'

not sure - again just going off the docs - if this is working for y'all great!

Reply:
Yeah I got the get route from the docs as well specifically for google_oauth2. It seems to be routing correctly in our tests and before the localhost crashes (no BE calls), again might be something that we have to verify once we can create and find users through the backend.